playwright_test/tests-examples/demo-todo-app.spec.js

```diff
@@ -0,0 +1,449 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
```

Comment:
is this example from playwright?
is it required?

Reply:
have deleted it.